src/backend/joanie/tests/core/test_models_enrollment.py

```diff
@@ -157,6 +159,46 @@ def test_models_enrollment_unique_opened_course_run_per_course_and_user(self):
             str(context.exception),
         )
 
+    def test_models_enrollment_not_unique_course_run_per_course_and_user(self):
+        """
+        A user can have multiple enrollments for a same course from the moment only one
+        is currently opened.
+        """
+        user = factories.UserFactory()
+        course = factories.CourseFactory()
+
+        with time_machine.travel(self.now - timedelta(days=7)):
```

Comment:
you can simply mock timezone, it's what we usually do in other projects
now = datetime(2018, 8, 8, tzinfo=timezone.utc)
with mock.patch.object(timezone, "now", return_value=now), mock.patch(
            "datetime.datetime"
        ) as mock_dt: